### PR TITLE
Circles core

### PR DESCRIPTION
**Pull request by thought2**

### CODE REVIEW

File: materialized/spago-packages.nix [hunk 1]
```diff
@@ -895,11 +895,11 @@ let
 
     "stadium" = pkgs.stdenv.mkDerivation {
         name = "stadium";
-        version = "fe351bdcc311667032b04ed1bbd2c9a7f56bdabf";
```
Comment (thought2) [inline materialized/spago-packages.nix:898]: ok, just saw that you worked on purescript-stadium. nice!

File: nix/pkgs/ts.nix [hunk 2]
```diff
@@ -75,46 +75,73 @@
 
   builds = {
     storybook = pkgs.runCommand "storybook" { buildInputs = [ pkgs.yarn ]; } ''
-      tmp=`mktemp -d`
 
+  ############ Create clean ts node package in temp dir
```
Comment (thought2) [inline nix/pkgs/ts.nix:79]: much better with the comments!

Comment (nmauersberg) [reply nix/pkgs/ts.nix:79]: Yeah, I think the next step could be, to extract some of these steps to their own functions.

File: pkgs/ts/storybook/package.json [hunk 3]
```diff
@@ -13,14 +13,14 @@
   },
   "dependencies": {
     "@storybook/addon-docs": "^6.4.18",
+    "assets": "*",
     "babel-preset-react-app": "^10.0.1",
     "circles": "*",
-    "assets": "*",
```
[the remark this answers is not in the document]
Comment (nmauersberg) [reply pkgs/ts/storybook/package.json:18]: It was just reordered by yarn

File: nix/pkgs/ts.nix (hunk 2 above)
Comment (thought2) [reply nix/pkgs/ts.nix:79]: or better tackle the root of the problem...

**Pull request merged**